### DISCUSSION TITLE
PostgreSQLSeverity.error 42P01: relation "serverpod_email_auth" does not exist

### DISCUSSION THREAD

**Discussion by andriyndev** [serverpod/serverpod#886] [Q&A] [answered]
From what I understand, the error is because the authentication-related tables are not present in the database. But how to create them? The tables are described in the `serverpod_auth_server` package, so we don't have the corresponding `tables.pgsql` file.

**Comment by Isakdl** [comment, marked as answer]
You can find the database setup sql file for serverpod_auth here: https://github.com/serverpod/serverpod/blob/main/modules/serverpod_auth/serverpod_auth_server/generated/tables.pgsql

How to add a module:
https://docs.serverpod.dev/concepts/modules

Hope this helps :)

**Reply by andriyndev** [reply]
Thank you. However, it'd be great to have an easier way to do this. For example, on `serverpod generate` copy the SQL files to the project's directory. Otherwise it's not very convenient and is quite confusing to look for a file on GitHub to use a standard module.

**Reply by vlidholt** [reply]
We are fixing this for version 1.2! :)
